contracts/dao/PancakeStaking.sol

```diff
@@ -169,6 +189,25 @@ contract PancakeStaking is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         emit RegisterPool(lpToken, poolAddress, distributor);
     }
 
+    function registerUsdtPool(address v2wrapper, address distributor) external onlyOwner {
```

Comment:
It's better use `registerPool`, we can support other's pool in the future.

Reply:
`registerPool` method already there. `registerUsdtPool` is added to differentiate with our exiting Lista-Pancake StableSwap LPs because the key in the `pools[lpToken]` is occupied.

Do you mean not to hard code the USDT addresses inside the function?

Reply:
Removed this function as design changes in ce3c9391542eaf55ff15ad8cf85864c4efb33b8d